sdk/iot/Azure.Iot.Hub.Service/tests/TestSettings.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    public class TestSettings
+    {
+        public const string IotHubEnvironmentVariablesPrefix = "IOTHUB";
```

Comment:
This should match the service directory name, so just "IOT"

Reply:
I see!! Will change. Thanks.